test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -451,7 +451,7 @@ public void ApplyConfiguration_Queues()
             scriptConfig.HostConfig.HostConfigMetadata = config;
             TraceWriter traceWriter = new TestTraceWriter(TraceLevel.Verbose);
 
-            scriptConfig.HostConfig.CreateMetadataProvider(); // will cause extensions to initialize and consume config metadata.
+            new JobHost(scriptConfig.HostConfig).CreateMetadataProvider(); // will cause extensions to initialize and consume config metadata.
```

Comment:
[https://github.com/Azure/azure-webjobs-sdk/issues/1201](https://github.com/Azure/azure-webjobs-sdk/issues/1201)

---
In reply to: [123674427](https://github.com/Azure/azure-webjobs-sdk-script/pull/1615#discussion_r123674427) [](ancestors = 123674427)